analyticsexample/src/main/java/com/bitmovin/exoplayeranalyticsexample/MainActivity.java

```diff
@@ -94,6 +94,7 @@ private void createPlayer() {
             bitmovinAnalyticsConfig.setPath("/vod/new/");
             bitmovinAnalyticsConfig.setHeartbeatInterval(59700);
             bitmovinAnalyticsConfig.setIsLive(false);
+            bitmovinAnalyticsConfig.setResendDataOnHttpTimeout(true);
```

Comment:
I think the placement of this setting should probably change to be moved to the `CollectorConfig`